src/wasmv1_execution/abi/abis.rs

```diff
@@ -152,6 +152,9 @@ fn abi_call(store_env: FunctionEnvMut<ABIEnv>, arg_offset: i32) -> Result<i32, W
             let remaining_gas = handler.get_remaining_gas();
             let interface = handler.exec_env.get_interface();
             let module = helper_get_module(interface, bytecode, remaining_gas)?;
+            interface.increment_recursion_counter().map_err(|e| {
```

Comment:
double-counts with the `init_call_wasmv1`

Reply:
removed the one in init_call_wasmv1